whobpyt/data/generators.py

```diff
@@ -93,4 +92,51 @@ def gen_cube(device = torch.device("cpu")):
     
     return {"SC" : SC_mtx_norm, "dist" : dist_mtx, "LF" : LF_Norm, "Source FC" : SC_mtx_norm, "Channel FC" : LF_Con_Mtx}
     
-    
\ No newline at end of file
+
+def syntheticSC(numRegions, seed = None, maxConDist = 50):

```

Comment:
The docstring is not in standard format and cannot be parsed by the sphinx documentation maker. I'll add a commit to fix this.